test/index.spec.js

```diff
@@ -22,67 +24,103 @@ describe("babel-preset-env", () => {
-      assert.deepEqual(babelPresetEnv.getTargets({
-        electron: 1.4,
-        chrome: 50
-      }), {
-        chrome: 50
-      });
-    });
-
-    it("should overwrite Chrome number if Electron version is older", function() {
-      assert.deepEqual(babelPresetEnv.getTargets({
-        electron: 1.0,
-        chrome: 50
-      }), {
-        chrome: 49
+      it("should preserve lower Chrome number if Electron version is more recent", function() {
+        assert.deepEqual(babelPresetEnv.getTargets({
+          electron: 1.4,
+          chrome: 50
+        }), {
+          chrome: 50
+        });
       });
-    });
 
-    Object.keys(electronToChromiumData).forEach((electronVersion) => {
-      it(`"should work for Electron: ${electronVersion}`, function() {
+      it("should overwrite Chrome number if Electron version is older", function() {
         assert.deepEqual(babelPresetEnv.getTargets({
-          electron: electronVersion
+          electron: 1.0,
+          chrome: 50
         }), {
-          chrome: electronToChromiumData[electronVersion]
+          chrome: 49
         });
       });
-    });
 
-    it("should error if electron version is invalid", () => {
-      const fixtures = [
-        "0.19",
-        0.19,
-        999,
-        "999",
-      ];
+      Object.keys(electronVersions).forEach((electronVersion) => {
+        it(`"should work for Electron: ${electronVersion}`, function() {
+          assert.deepEqual(babelPresetEnv.getTargets({
+            electron: electronVersion
+          }), {
+            chrome: electronVersions[electronVersion]
+          });
+        });
+      });
+
+      it("should error if electron version is invalid", () => {
+        const fixtures = [
+          "0.19",
+          0.19,
+          999,
+          "999",
+        ];
+
+        fixtures.forEach((electronVersion) => {
+          assert.throws(() => {
+            babelPresetEnv.getTargets({
+              electron: electronVersion,
+            });
+          }, Error);
+        });
+      });
+    });
 
-      fixtures.forEach((electronVersion) => {
+    describe("`current` + Electron not installed", () => {
+      it("should error", () => {
         assert.throws(() => {
           babelPresetEnv.getTargets({
-            electron: electronVersion,
+            electron: "current",
           });
         }, Error);
       });
     });
+
+    describe("`current`", () => {
+      const electronDir = rootPath + "/node_modules/electron";
+
+      before(() => {
+        const source = rootPath + "/test/mocks/electron.package.json";
+        const destination = electronDir + "/package.json";
+        if (fs.existsSync(electronDir)) fs.removeSync(electronDir);
+        fs.mkdirSync(electronDir);
+        fs.copySync(source, destination);
+      });
+
+      after(() => {
+        fs.remove(electronDir);
+      });
+
+      it("should return the locally installed Electron version", function() {
```

Comment:
That diff isn't too clear, but I'm also testing for that [here](https://github.com/babel/babel-preset-env/pull/191/files#diff-faa503f61f8addeb603095cc2f40c82eR91)